### PR TITLE
Test/create db tests

### PR DESCRIPTION
**Pull request by dshelters**
Creates tests for the queries used for the rooms and users tables. Also standardizes the add queries to require an object when adding data to any table. Add queries will also auto-update with the current time in the table as well.

### CODE REVIEW

File: spec/database-spec/simpleQueries.spec.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  it('should have one entry populated', () => (
+    db.users.all().then((results) => {
+      expect(results.length).toEqual(1);
+    })
+  ));
+
+  it('return an id of 2 when a second user is added', () => (
+    db.users.add({
+      name: 'Multipass',
+      email: 'Multi-pass',
```
Comment (misteryeo) [inline spec/database-spec/simpleQueries.spec.js]: I don't think this matters, but why isn't this actually an email?

File: database/queries/messages_code/create.sql [hunk 2]
```diff
@@ -3,8 +3,8 @@ CREATE TABLE IF NOT EXISTS messages_code
   id SERIAL PRIMARY KEY,
   user1_Id INT,
   user2_Id INT,
-  room_Id VARCHAR(40) UNIQUE NOT NULL,
+  room_Id VARCHAR(40) NOT NULL,
```
Comment (dshelters) [inline database/queries/messages_code/create.sql]: We definitely do want unique room ids, just not on this table. Room Id's will be unique on the rooms table, but the messages_code join table will have potentially have an unlimited number of messages sent on the same room, and we would only be able to save 1 message per room if we force the id to be unique in this table.

File: spec/database-spec/simpleQueries.spec.js (hunk 1 above)
Comment (dshelters) [inline spec/database-spec/simpleQueries.spec.js]: No reason, except that I love The Fifth Element and putting an '@' and a '.com' would have ruined the multipass flow.

Comment (jefQuery) [inline spec/database-spec/simpleQueries.spec.js]: Mul-ti-pass.

**Comment by jefQuery** [comment]
[Debug why code won't run with code press (fb OAuth)](https://trello.com/c/gHolmjuW/105-debug-why-code-won-t-run-with-code-press-fb-oauth)

**Pull request merged**